spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/data/SparkParquetReaders.java

```diff
@@ -373,41 +371,6 @@ public Decimal read(Decimal ignored) {
     }
   }
 
-  private static class TimestampMillisReader extends UnboxedReader<Long> {
```

Comment:
Only removed for latest version of spark, thinking when we deprecate the older versions, this code will be removed. 
Spark-3.3 is deprecated. Spark-3.4 Should I handle it?

Reply:
I think this is fine. No need to change older Spark versions.